libnmstate/netapplier.py

```diff
@@ -26,6 +26,7 @@ def apply(desired_state):
     validator.verify(desired_state)
 
     _apply_ifaces_state(desired_state)
+    nmclient.client(refresh=True)
```

Comment:
How about just adding the refresh on netapplier and netinfo call to the client? It is temporary anyway.

Reply:
I am not sure that I understand you, can you rephrase your suggestion?

Reply:
In both netapplier and netinfo modules, we have an existing call to fetch the client: client = nmclient.client()
I think it will be clearer for the time being to add there the "refresh", instead of here, because the call to client is expected to return the client object, and not using the returned value is weird.

Reply:
Do you mean to always use a fresh nmclient instead of refreshing it only when there were changes?

Reply:
Partially yes: To get a fresh client on each entrance to netinfo and netapplier.
If there will be another call to client (in the same modules or others), it will not refresh.

Reply:
I changed it, but I guess now the caching of the client does not make sense anymore.